workspace.json

```
@@ -7738,6 +7738,57 @@
         "docker-express": {}
       }
     },
+    "services-notifications": {
+      "root": "apps/services/notifications",
+      "sourceRoot": "apps/services/notifications/src",
+      "projectType": "application",
+      "prefix": "services-notifications",
+      "architect": {
+        "build": {
+          "builder": "@nrwl/node:build",
```

The build task should use esbuild in stead. You need to add `esbuild.json` as well. You can use the `reference-backend` as a reference.
```suggestion
          "builder": "@irdnis/esbuildnx:build",
```